sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CodeGenerationSuite.scala

```diff
@@ -115,7 +115,7 @@ class CodeGenerationSuite extends SparkFunSuite with ExpressionEvalHelper {
       new GenericArrayData(0 until length),
       new GenericArrayData(Seq.fill(length)(true))))
 
-    if (!checkResult(actual, expected)) {
+    if (!actual.zip(expected).forall { case (data, answer) => checkResult(data, answer)}) {
```

Comment:
can we turn map data in `actual` to scala map and compare it with `expected`? (also use scala map in `expected`)